style.css

```diff
@@ -0,0 +1,157 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  list-style-type: none;
+}
+
+/* navbar styles */
+.navbar {
+  width: 100%;
+  height: auto;
+  padding: 10px 12px;
+  left: 0%;
+  right: 0%;
+  background: #3c3a39;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: row;
+  position: absolute;
+  flex-wrap: wrap;
+  margin-right: 100px;
+}
+
+.navbar h4 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 10px;
+  font-size: 19px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: white;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 10px;
+}
+
+.navbar img {
+  display: flex;
+  right: 27.67%;
+  left: 16%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid #fff;
+  margin-bottom: 3px;
+  flex-grow: 0;
+  order: 0;
+  flex-direction: row;
+}
+
+#navbar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  color: white;
+  margin-right: 20px;
+  margin-bottom: 12px;
+  float: right;
+  margin-top: 40px;
+}
+
+#navbar img {
+  display: flex;
+  right: 27%;
+  left: 16.67%;
+  height: 2px;
+  width: 17px;
+  border: 1px solid white;
+  margin-bottom: 3px;
+}
+
+/* headline styles */
+.headline {
+  overflow: hidden;
+  background-image: url(Images/Group.png);
```

Comment:
- [ ] I see that you tried to apply the background image, but like the social link list, there is no image in the repository to load. Kindly upload the images to this repository so the app can load.